src/spec/cli.cr

```diff
@@ -9,12 +9,12 @@ module Spec
       property formatters = Array(Spec::Formatter).new
       property locations = Array({String, Int32}).new
 
-      property! default_formatter : Spec::Formatter
-      property! fail_fast : Bool
-      property! line : Int32
-      property! no_color : Bool
-      property! pattern : String
-      property! slowest : Int32
+      property? default_formatter : Spec::Formatter? = nil
```

Comment:
No need for nilable postfix `?`, since it's added automatically.
Default `nil` assignments are also redundant.
ditto all below